tests/_fixtures.py

```diff
@@ -97,6 +100,34 @@ def style_image(image_small_2):
     return image_small_2
 
 
+@pytest.fixture
+def guides(target_image):
+    return {
```

Comment:
It would be better to create a bool "guide" first and than invert it for the second region. With this you can be sure that the whole canvas is filled and the regions do not overlap.